File: evidence/generated-systems/angular/hr-on-angular/src/app/models/BankAccount.js

// Define collection and schema for BankAccount
export interface BankAccount {
    accountHolder:
	type : string
    bankName:
	type : string
    iban:
	type : string
    bic:
	type : string
    accountNumber:
	type : string
    routingNumber:
	type : string
#
    collection: 'bankAccounts'
}

File: evidence/generated-systems/golang/hr-on-golang/hr-on-golang/internal/model/BankAccount.go
package model

import (
    "gorm.io/gorm"
)

//==============================================================
// BankAccount Declaration
//==============================================================
type BankAccount struct {
    gorm.Model
     AccountHolder                                    string
    BankName                                    string
    Iban                                    string
    Bic                                    string
    AccountNumber                                    string
    RoutingNumber                                    string

// parent associations as their child

}

